traitsui/tests/test__tools.py

```diff
@@ -87,6 +89,14 @@ class TestProcessEventsRepeated(unittest.TestCase):
     def test_qt_process_events_process_all(self):
         from pyface.qt import QtCore
 
+        if QtCore.__version_info__ < (5, 0, 0) and is_mac_os:
```

Comment:
any reason why we can't use the `is_qt4` util? (i dont remember now if that's in pyface or traitsui or both)

Reply:
and if we use the `is_qt4` util, we can simply move this `skipTest` from inside the method to a decorator.

Reply:
`is_qt4` is from pyface. I could import `is_qt4` from pyface and there it does just mean Qt4 only. But then in `traitsui.tests._tools` , the `is_current_toolkit_qt4` actually means both Qt4 and Qt5 and is used in a lot of places in traitsui. Currently, all `is_whatever_qt4` in traitsui means both Qt4 and Qt5. We need to sort this out. But before that I want to avoid mixing more meaning for these `is_whatever_qt4` within traitsui.

Reply:
#978